packages/content-conformance/src/test/utils.ts

```diff
@@ -20,11 +20,12 @@ export function getFixturePath(fixtureName: string) {
 function makeTestFileAndContext(
   rule: ConformanceRuleBase,
   fixture: VFileCompatible,
-  contentFiles: VFileCompatible[] = []
+  contentFiles: VFileCompatible[] = [],
+  dataFiles: VFileCompatible[] = []
 ) {
   let file
   const context: ConformanceRuleContext = {
-    dataFiles: [],
+    dataFiles: dataFiles.map((e) => new DataFile(e)),
```

Comment:
This adds support for passing `dataFiles` in with the test case, like @thiskevinwang did for `contentFiles`